build.sbt

```diff
@@ -322,6 +335,12 @@ lazy val skipReleaseSettings = Seq(
   publish / skip := true
 )
 
+lazy val javaOnlyReleaseSettings = releaseSettings ++ Seq(
+  crossPaths := false, // drop off Scala suffix from artifact names
+  publishArtifact := scalaBinaryVersion.value == "2.12", // only publish once
```

Comment:
why is this hard coded to 2.12? isnt there a scala version string defined as variable somewhere in the sbt?

Reply:
`scala212 = 2.12.14`. We need `2.12` not `2.12.x`. I can make this into a final variable, too. Im easy.

Reply:
made it into a variable